commands.json

```diff
@@ -1406,20 +1474,21 @@
         "type": "integer",
         "optional": true,
         "token": "DIALECT",
-        "since": "2.4.3"
+        "since": "2.4.3",
```

Comment:
Nitpick - are we considering `since` when generating the command info?

Reply:
Yes:
![image](https://github.com/user-attachments/assets/9f220d06-fbcf-4ada-9bdd-e088c927ec7d)